### DISCUSSION TITLE
Setting Timezone

### DISCUSSION THREAD

**Discussion by jordanstudioroot** [jrnl-org/jrnl#1606] [Q&A]
I have just started using this tool, and I like everything about it except for one thing. I can't figure out how to get the timezone to format in PDT/PST.

I'm running Ubuntu in WSL2.

cat /etc/*release

DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
NAME="Ubuntu"
VERSION="20.04.4 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.4 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
Red Hat Linux release 5

From what I can see in the source code, each entry object creates its timestamp on creation using UTC (since there is no timezone arg provided). 

The timeformat parameter in the config yaml doesn't support timezone formatting, according to the docs.

Aliases like -today don't seem to be sensitive to the current timezone and manual timezone offsets. For example, if I set timeformat to %Y-%m-%d %H:%M:%S -0700, -today will still only return entries from yesterday at 6 AM PDT.

Is there something obvious I am missing? I've searched the issues and discussion for timezone related things and didn't find anything relevant to my problem.

**Comment by micahellison** [comment]
Thanks for filing this, @jordanstudioroot. I was able to reproduce the problem so it's certainly a bug, though I don't know where the core problem is -- it could be in jrnl, or in a library jrnl uses, or in something that's lost in translation as it crosses from Windows to Ubuntu to the various Python modules that know about time.

Whatever the case, I filed it in #1607 and will take a look at it later.

In the meantime, if you're still interested in using jrnl on your Windows machine, it should work as expected in the normal Windows environment. That's how I run it, with Python installed via winget, then running `pip install pipx` then `pipx install jrnl`. You might have to close and re-open your terminal in between if you use this approach.
